fuzzing/fuzz.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+root=$(cd $(dirname "$0")/.. && pwd)
+
+if command -v echidna-test; then
+  fuzz () {
+    echidna-test ${root} \
+      --config ${root}/fuzzing/echidna.yaml \
+      --corpus-dir ${root}/fuzzing/corpus \
+      --contract $1
+  }
+else
+  fuzz () {
+    docker run \
+      --rm \
+      -v ${root}:/src ghcr.io/crytic/echidna/echidna \
+      bash -c \
+        "cd /src && echidna-test . \
+          --config fuzzing/echidna.yaml \
+          --corpus-dir fuzzing/corpus \
+          --crytic-args --ignore-compile \
```

Comment:
Noticed that `--crytic-args --ignore-compile` flags were not in the non-docker command. Just checking that these weren't added inadvertently.